sdk/servicebus/azure-messaging-servicebus-track2-perf/src/main/java/com/azure/messaging/servicebus/perf/BatchSendTest.java

```diff
@@ -0,0 +1,76 @@
+package com.azure.messaging.servicebus.perf;
+
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusMessage;
+import com.azure.messaging.servicebus.ServiceBusSenderAsyncClient;
+import com.azure.messaging.servicebus.ServiceBusSenderClient;
+import com.azure.perf.test.core.BatchPerfTest;
+import com.azure.perf.test.core.TestDataCreationHelper;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class BatchSendTest extends BatchPerfTest<ServiceBusStressOptions> {
```

Comment:
Could you add description to each test, about what feature are you testing, what be expected input/output value?